media-video/handbrake/patches/handbrake-1.0.7-4.patchset

```diff
@@ -0,0 +1,227 @@
+From: Ken Mays <kmays2000@gmail.com>
+Date: Fri Aug 10 13:01:42 GMT 2018 
+Subject: fix for FFmpeg 4.x dependency
+
+
+--- HandBrake-1.0.7/libhb/encavcodec.c.old	2018-08-10 11:50:45.138674176 +0000
```

Comment:
Are you sure it applies cleanly?
Please check it again and fix it.

Reply:
Please, try to build stuff with haikuporter before you make any PR.
This is definetly wrong as the patches shouldn't change the filename.

Reply:
Yes, I fixed the recipe and the patch. This is what I used to build the handbrake 1,0,7-4 package in haikuporter. I had changed the filename just for review.